homeassistant/components/binary_sensor/.translations/it.json

```diff
@@ -59,11 +59,11 @@
             "moving": "{entity_name} ha iniziato a muoversi",
             "no_gas": "{entity_name} ha smesso la rilevazione di gas",
             "no_light": "{entity_name} smesso il rilevamento di luce",
-            "no_motion": "{nome_entit\u00e0} ha smesso di rilevare il movimento",
-            "no_problem": "{nome_entit\u00e0} ha smesso di rilevare un problema",
+            "no_motion": "{entity_name} ha smesso di rilevare il movimento",
+            "no_problem": "{entity_name} ha smesso di rilevare un problema",
             "no_smoke": "{entity_name} ha smesso la rilevazione di fumo",
-            "no_sound": "{nome_entit\u00e0} ha smesso di rilevare il suono",
-            "no_vibration": "{nome_entit\u00e0} ha smesso di rilevare le vibrazioni",
+            "no_sound": "{entity_name} ha smesso di rilevare il suono",
+            "no_vibration": "{entity_name} ha smesso di rilevare le vibrazioni",
```

Comment:
`.translations` folder is overridden when there is the `[ci skip] Translation update` commit by `HomeAssistant Azure`

So you should use [Localise](https://lokalise.com/projects) to translate HA to all components you want by joining the team [here](https://developers.home-assistant.io/docs/en/internationalization_translation.html) (with the "Join the backend translation team" link).

Closing the PR.

Reply:
I'm very sorry. I spotted the problem here and i thought was fine to fix it here. I've updated on Lokalise now.  Sorry again

Reply:
No worries :wink: